contrib/win32/openssh/config.h.vs

```diff
@@ -1565,7 +1565,7 @@
 #define _PATH_PASSWD_PROG "/usr/bin/passwd"
 
 /* Specify location of ssh.pid */
-#define _PATH_SSH_PIDDIR "."
+/* #undef _PATH_SSH_PIDDIR */
```

Comment:
why do we need this as we define at line 1696.

Reply:
I believe this is only created by sshd deamon process only?

sshd process spawned by sshd deamon will not write to this location?

Reply:
Right. It shouldn't and it won't.